src/database/stores/RethinkDbGuildStore.ts

```diff
@@ -725,12 +725,13 @@ export class RethinkDbGuildStore implements GuildStore {
         return true;
     }
 
-    public async setLogIgnores(guildId: string, userIds: string[]): Promise<boolean> {
+    public async setLogIgnores(guildId: string, userIds: string[], ignore: boolean): Promise<boolean> {
         const guild = await this.#table.get(guildId);
         if (guild === undefined)
             return false;
 
-        if (!await this.#table.update(guildId, r => ({ logIgnore: r('logIgnore').default([]).setUnion(userIds) })))
+        const method = ignore ? 'setUnion' : 'setDifference';
+        if (!await this.#table.update(guildId, r => ({ logIgnore: r('logIgnore').default([])[method](userIds) })))
```

Comment:
any possibility this line can be made easier to understand? 😳

Reply:
I could split it onto multiple lines? But its rethink, its not the most elegant of syntaxes in general

Reply:
Added comments